### PR TITLE
[MOD-2183] iOS: Support statusBarStyle from parent window

### PR DESCRIPTION
**Pull request by hansemannn**
JIRA: https://jira.appcelerator.org/browse/MOD-2183

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,16 +1,19 @@
...
+   - "MODULE_NAME=ti.barcode"
 before_install:
     - MODULE_ROOT=$PWD
 install:
     - cd $MODULE_ROOT
-    - curl -o install.sh https://raw.githubusercontent.com/appcelerator-modules/ci/master/travis/install.sh
-    - source install.sh 
-script: 
+    - curl -o install.sh https://raw.githubusercontent.com/appcelerator-modules/ci/master/travis/install.sh #change this to appcelerator-modules once PR has been merged
+    - source install.sh -s "--branch 5_1_X"
```
Comment (ashcoding) [inline .travis.yml]: I changed this to `- source install.sh -a 22 -s "5.1.2.v20151216190036" ` in https://github.com/appcelerator-modules/ti.barcode/pull/33

The Android part is passing with that change. But overall it's still failing.

Comment (hansemannn) [inline .travis.yml]: Hey Ash, thanks! I updated the `VALID_ARCHS` property to pass the 64bit-checks. Let's see if it passes now. Would be great to be re-validated by you.

**Comment by ashcoding** [comment]
As mentioned above, could you try changing line 13 to `- source install.sh -a 22 -s "5.1.2.v20151216190036"` to fix the android portion. I tried it here https://github.com/appcelerator-modules/ti.barcode/pull/33 and it passes the Android part.

**Comment by hansemannn** [comment]
Mhh I tried to prevent hardcoding the actual build version, but will check.

**Comment by ashcoding** [comment]
With that changed, the android part is now passing. :+1:  But overall, it's still failing. I don't really see where it's failing except for here https://travis-ci.org/appcelerator-modules/ti.barcode/builds/100788515#L4031-L4040
Is it Xcode related?

**Comment by cheekiatng** [comment]
Yes it's Xcode related. caused by this line https://github.com/hansemannn/ti.barcode/blob/master/ios/build.py#L239 when trying to package. It says there was no input file specified here https://github.com/appcelerator-modules/ti.barcode/blob/master/ios/build.py#L192
you might want to make reference to build.py in ti.facebook to see if it gives any clues.

**Comment by hansemannn** [comment]
@cheekiatng @ashcoding Updated the build script with the current one from ti.facebook, the build succeeds now.

**Comment by sriks** [comment]
Code reviewed, approved.

**Pull request merged**